cw_core/lib/wallet_keys_file.dart

```diff
@@ -0,0 +1,76 @@
+import 'dart:convert';
+import 'dart:io';
+
+import 'package:cw_core/pathForWallet.dart';
+import 'package:cw_core/utils/file.dart';
+import 'package:cw_core/wallet_type.dart';
+
+mixin WalletKeysFile {
+  // this needs to be overridden
+  Future<String> makePath() => throw UnimplementedError();
+
+  // this needs to be overridden
+  WalletKeysData get walletKeysData => throw UnimplementedError();
```

Comment:
if needs to be overridden, then why give them a body and throw an exception, just make them abstract so they give error when using them
<img width="377" alt="Screenshot 2024-08-08 at 1 27 58 AM" src="https://github.com/user-attachments/assets/98b17e96-ac15-4f9c-838e-b21edf9919d2">